dom.bs

```diff
@@ -1138,15 +1137,22 @@ for discussion).
   <p class="note"><var>legacy target override flag</var> is only used by HTML and only when
   <var>target</var> is a {{Window}} object.
 
- <li>Let <var>relatedTarget</var> be the result of <a>retargeting</a> <var>event</var>'s
- <a for=Event>relatedTarget</a> against <var>target</var> if <var>event</var>'s
- <a for=Event>relatedTarget</a> is non-null, and null otherwise.
+ <li><p>Let <var>clearTargetsPostDispatch</var> be true if either <var>target</var> or one of
+ <var>event</var>'s <a for=Event>relatedTargets</a> is a <a>node</a> and its <a for=tree>root</a> is
+ a <a for=/>shadow root</a>, and false otherwise.
+
+ <li><p>Let <var>relatedTargets</var> be a new <a for=/>list</a>.
+
+ <li><p><a for=list>For each</a> <var>relatedTarget</var> of <var>event</var>'s
+ <a for=Event>relatedTargets</a>, <a for=list>append</a> the result of <a>retargeting</a>
+ <var>relatedTarget</var> against <var>target</var> to <var>relatedTargets</var>.
 
- <li><p>If <var>target</var> is <var>relatedTarget</var> and <var>target</var> is not
- <var>event</var>'s <a for=Event>relatedTarget</a>, then return true.
+ <li><p>If <var>relatedTargets</var> <a for=list>contains</a> <var>target</var> and
```

Comment:
I doubt this is quite right in Touch case. Need to think this case some more.
I have some such case in mind where one of the relatedTargets does contain target before retargeting and then retargeting makes also some other relatedTarget to be target.

Reply:
It seems https://github.com/w3c/web-platform-tests/blob/master/touch-events/touch-retargeting.html can be extended somehow for more test coverage for touch. It's not entirely clear to me how that works given that on desktop the events are not really exposed.